pkg/metrics/api/api.go

```diff
@@ -0,0 +1,456 @@
+package api
+
+import (
+	"math"
+	"net/http"
+	"os"
+
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/openshift/installer/pkg/metrics/builder"
+	"github.com/openshift/installer/pkg/metrics/pushclient"
+)
+
+var (
+	// INVOCATION METRIC
+
+	// ClusterInstallationCreateJobName is the variable that stores the metric name for the create cluster details.
+	ClusterInstallationCreateJobName = "cluster_installation_create"
+	// ClusterInstallationIgnitionJobName is the variable that stores the metric name for the create ignition details.
+	ClusterInstallationIgnitionJobName = "cluster_installation_ignition"
```

Comment:
I don't think `ClusterInstallation` is a useful prefix in the context of installer, so perhaps a pattern like, in this case:

```go
CreateIgnitionMetric = "cluster_installation_ignition"
```

or 

```go
CreateIgnitionMetricName = "cluster_installation_ignition"
```